systest/parameters/bignet/smesher.json

```diff
@@ -37,6 +37,6 @@
         "post-k1": 273,
         "post-k2": 300,
         "post-k3": 100,
-        "post-k2pow-difficulty": 469112881707866048
+        "post-pow-difficulty": 469112881707866048
```

Comment:
These need to be updated too.

Reply:
why do we need to change that param?

Reply:
Because difficulty changed from an uint64 to an uint256 (represented as 64 bytes)